elasticsearch/mapping/index_settings.json

```diff
@@ -74,7 +76,16 @@
                     }
                 },
                 "misc": {
-                    "type": "object"
```

Comment:
Hi @rsmith013, I see that you have removed line 77 of the mapping that used to identify the "type" of the "misc" section as "object".
Please can you confirm this was planned - and has no undesirable consequences. Thanks